apache/app.mako-dot-conf

```diff
@@ -35,7 +35,7 @@ RewriteRule ^${apache_base_path}/[0-9]+/(img|lib|style|locales)(.*) ${apache_bas
 </LocationMatch>
 
 # Sitemaps (we are using version here to get a cached version from backend)
-RewriteRule ^${apache_base_path}/sitemap_(.*)\.xml http:${api_url}/${version}/sitemap?content=$1 [P]
+RewriteRule ^${apache_base_path}/sitemap_(.*)\.xml https:${api_url}/${version}/sitemap?content=$1 [P]
```

Comment:
I do not understand why...

https://mf-chsdi3.dev.bgdi.ch/ltpoa/1430399263/sitemap?content=base exists...

Maybe some apache trick not liking going from http to https.